components/input-number/index.en-US.md

```diff
@@ -20,6 +20,8 @@ When a numeric value needs to be provided.
 | autoFocus | If get focus when component mounted | boolean | false | - |
 | bordered | Whether has border style | boolean | true | 4.12.0 |
 | controls | Whether to show `+-` controls | boolean | true | 4.17.0 |
+| customUpIcon | The custom icon displayed instead of the default up arrow icon | ReactNode | - |  |
```

Comment:
Add a demo?

Reply:
Got it, working on that.

Reply:
@xrkffgg I just added a demo, I translate the words to Chinese simplified with google translate, feel free to update that if it's wrong.